demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -247,6 +247,13 @@ def _is_pack_meta_file_structure_valid(self):
                                            self.pack_meta_file):
                             return False
 
+            # if the field 'certification' exists, check that its value is set to 'certified' or 'verified'
+            certification = metadata.get(PACK_METADATA_CERTIFICATION)
+            if certification and certification not in ['certified', 'verified']:
```

Comment:
You can create a constant - "ALLOWED_CERTIFICATION_VALUES" = ['certified', 'verified']
So if want to add/remove one of them in the future it will be easier.
And then ask:
`if certification and certification not in ALLOWED_CERTIFICATION_VALUES:`